src/agent/script/win64/libfuzzer-coverage/DumpCounters.js

```diff
@@ -98,8 +98,12 @@ function dumpCounters(results_dir, should_disable_sympath) {
     // Disable FCE from sanitizers.
     execute("sxd av");
 
-    // Run to exit break in `ntdll!NtTerminateProcess`.
-    execute("g");
+    // run until LLVMFuzzerTestOneInput
```

Comment:
```suggestion
    // Run until `LLVMFuzzerTestOneInput()`.
    // This makes us unlikely to have unloaded any modules that the user dynamically loaded,
    // and so we will still be able to dump the coverage tables for those modules.
```

Reply:
I believe this newly narrows the applicability of this script to libFuzzer targets only, which wasn't the case previously (or at least originally). In practice, that's just fine.